composer.json

```diff
@@ -12,10 +12,10 @@
     "preferred-install": "dist"
   },
   "require": {
+    "dosomething/gateway": "^1.0.0-rc14",
     "stripe/stripe-php": "^1.18.0",
-    "zendesk/zendesk_api_client_php": "^1.2.0",
-    "tightenco/collect": "^5.2",
-    "dosomething/gateway": "^1.0.0-rc14"
+    "tightenco/collect": "^5.4",
```

Comment:
One of the methods I used for the collection was giving back really odd results. Turns out the method was updated and [fixed in v5.4](https://github.com/tightenco/collect/pull/32/files#diff-1f18a78060929ba4f93e96a4d21d022dL628).